integrationtests/tests/tests.rs

```diff
@@ -317,6 +317,7 @@ async fn drop_peers_who_contribute_bad_sigs() {
     assert!(fed.subset_peers(&[0, 1, 2]).has_dropped_peer(3));
 }
 
+/*
```

Comment:
@m1sterc001guy , why do we have to disable this test with the DB change?

Reply:
PS: disabling by comment is super easy to miss in reviews. I think we should use `#[ignore]` macro

```
#[tokio::test(flavor = "multi_thread")]
#[ignore]
async fn my_test_case() { ... }
```

Reply:
Oops apologies, didn't mean to keep that in there. I will revert.

Will use ignore in the future, thanks for the tip.

Reply:
Re-enable the test: https://github.com/fedimint/fedimint/pull/743

The test is stalling for me locally (with and without my change) but I haven't had time to dig into why.